hadoop-hdds/docs/content/interface/O3fs.md

```diff
@@ -59,7 +59,7 @@ Please add the following entry to the core-site.xml.
 </property>
 <property>
   <name>fs.defaultFS</name>
-  <value>o3fs://bucket.volume</value>
+  <value>o3fs://bucket.volume.OzoneServiceId</value>
```

Comment:
Can we add a some explanation here for OM HA cluster that ozone service id need to be specified, for non-ha it can be just o3fs://bucket.volume

Reply:
Thank @bharatviswa504  for review, can I change it to the following tip?
![image](https://user-images.githubusercontent.com/72794035/123400208-16f8fa00-d5d8-11eb-8b8f-4c580106e9f8.png)

Reply:
Looks good, what is the value you want to keep in fs.defaultFS?

Reply:
This is not a HA document, is it better to keep `o3fs://bucket.volume`?

Reply:
Yes, better to keep o3fs://bucket.volume.
And I like the Tip idea, which communicates what is needed.

In the Tip in addition to specifying ozone service id give an example URL also.